pkg/util/atomicfs/fsync_test.go

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package atomicfs
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateFile(t *testing.T) {
+	path := filepath.Join(t.TempDir(), "test")
+	wantData := "test1"
+	if err := CreateFile(path, strings.NewReader(wantData)); err != nil {
+		t.Errorf("CreateFile() error = %v, wantErr %v", err, false)
+	}
+	data, err := os.ReadFile(path)
+	require.NoError(t, err)
+
+	require.Equal(t, wantData, string(data))
+}
+
+func TestCreateFileAndMove(t *testing.T) {
+	tmpPath := filepath.Join(t.TempDir(), "test")
+	finalPath := filepath.Join(t.TempDir(), "testFinal")
+	wantData := "test1"
+	if err := CreateFileAndMove(tmpPath, finalPath, strings.NewReader(wantData)); err != nil {
+		t.Errorf("CreateFileAndMove() error = %v, wantErr %v", err, false)
+	}
```

Comment:
Is there a reason for using `t.Errorf` directly?

```suggestion
	err := CreateFileAndMove(tmpPath, finalPath, strings.NewReader(wantData))
	require.NoError(t, err)
```

Reply:
It must be a relic from the table test. Applying the suggestion.